programs/mango-v3-reimbursement/src/state/table.rs

```diff
@@ -0,0 +1,20 @@
+use std::mem::size_of;
+
+use anchor_lang::prelude::*;
+use static_assertions::const_assert_eq;
+
+#[account(zero_copy)]
+pub struct Table {
+    pub rows: [Row; 1], // TODO: change to 32000
+}
+const_assert_eq!(size_of::<Table>(), (32 + 8 * 16) * 1);
+const_assert_eq!(size_of::<Table>() % 8, 0);
+
+#[derive(Copy, Clone, AnchorSerialize, AnchorDeserialize)]
+#[repr(C)]
+pub struct Row {
+    pub owner: Pubkey,
+    pub balances: [u64; 16],
```

Comment:
I wonder if we should drop LUNA, COPE, ADA and reduce table size by 19%. Probably not though, the index confusion would be a potential source of bugs.